ballroom-beats/src/components/Navbar/Navbar.js

```diff
@@ -32,7 +32,7 @@ const AppNavigator = createStackNavigator({
 
 const styles = StyleSheet.create({
   bottomContainer: {
-    backgroundColor: '#FF3C6F',
+    backgroundColor: '#BD4413',
```

Comment:
@Asilo5 I love the colors you picked. But, maybe we can have a little brighter of a navbar since it's a game. I was messing around in coolors and used this #D1005A. It really pops and looks good with the green of the buttons.

Reply:
Thanks Allison! will add that on!